test/compress/functions.js

```diff
@@ -0,0 +1,6 @@
+non_ascii_function_identifier_name: {
+    input: {
+        function fooλ(δλ) {}
```

Comment:
Was hoping to exercise non-ascii names in AST_Function (function expression) in addition to AST_Defun (function declaration) which is why I suggested `(function λ(δλ) {})();`